src/TipBox.vue

```diff
@@ -151,6 +160,11 @@
         }
         return '';
       },
+      hasIcon() {
```

Comment:
Should we log a warning if both `no-icon` and `icon` is set? This might be helpful to help the user debug issues.

Similar discussion in https://github.com/MarkBind/vue-strap/pull/118.

Reply:
We should also note in the user guide that custom styles will take precedence over any other styling options (including the `no-*` attributes that we are adding here.

Reply:
![image](https://user-images.githubusercontent.com/27397021/74716883-56883700-526a-11ea-8eab-3fb7edf0bfaf.png)

![image](https://user-images.githubusercontent.com/27397021/74717036-a1a24a00-526a-11ea-9161-6f4067929a54.png)


Regarding documentation, I've done so 👌